.github/workflows/build.yml

```diff
@@ -69,11 +71,76 @@ jobs:
           version: tags/20240511.0
           file: firebase-windows-amd64.zip
 
-      - run: Expand-Archive -Path firebase-windows-amd64.zip -DestinationPath third_party; Rename-Item -Path third_party/firebase -NewName firebase-development
+      - run: |
+          Expand-Archive -Path firebase-windows-amd64.zip -DestinationPath third_party
+          Rename-Item -Path third_party/firebase -NewName firebase-development
         shell: powershell
 
       - name: Configure
-        run: cmake -B out -D CMAKE_BUILD_TYPE=Release -G Ninja -S ${{ github.workspace }} -D CMAKE_Swift_FLAGS="${{ matrix.options }}" -D SWIFT_FIREBASE_BUILD_EXAMPLES=NO
+        run: |
+          cmake -B out `
+                -D CMAKE_BUILD_TYPE=Release `
+                -G Ninja -S ${{ github.workspace }} `
+                -D CMAKE_Swift_FLAGS="${{ matrix.options }}" `
+                -D SWIFT_FIREBASE_BUILD_EXAMPLES=NO
+
+      - name: Build
+        run: cmake --build out
+
+  android:
+    runs-on: windows-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - branch: development
+            tag: DEVELOPMENT-SNAPSHOT-2024-07-09-a
+
+            abi: arm64-v8a
+            options: -sdk ${env:SDKROOT}\..\..\..\Android.platform\Developer\SDKs\Android.sdk -sysroot ${env:ANDROID_NDK_ROOT}\toolchains\llvm\prebuilt\windows-x86_64\sysroot -I${env:SDKROOT}\..\..\..\..\..\..\Toolchains\0.0.0+Asserts\usr\include -Xlinker -zdefs
+            target: aarch64-unknown-linux-android28
+
+    name: CMake (Android) - Swift ${{ matrix.tag }}
+
+    steps:
+      - uses: compnerd/gha-setup-vsdevenv@main
+
+      - uses: compnerd/gha-setup-swift@main
+        with:
+          # tag: ${{ matrix.tag }}
+          # branch: ${{ atrix.branch }}
```

Comment:
Do we need to keep these around?

Reply:
No, but I want to switch back ASAP. I am going to try to figure out how to get the Apple releases to include the SDKs so we can use official Swift releases for the Android CI as well.